packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -763,7 +763,7 @@ const DatePickerInput = (props) => {
       size="2"
       pattern="\d*"
       aria-required={required}
-      aria-label={intl.formatMessage({ id: 'Terra.datePicker.monthLabel' })}
+      aria-label={`${ariaLabel || ''} ${intl.formatMessage({ id: 'Terra.datePicker.monthLabel' })}`}
```

Comment:
Can we move up this condition to remove the extra space seen in jest snaps https://github.com/cerner/terra-framework/blob/8d4603715fe10fee9e460c580b7f7c9635d09565/packages/terra-date-picker/tests/jest/__snapshots__/DatePicker.test.jsx.snap#L550

Reply:
This condition has been added to handle the scenario when user doesn't pass the ariaLabel in such case SR reads as undefined if this condition has removed .

Reply:
If it doesnt get passed, then aria-label should be only set to the intl string directly. This will remove that single space.  Condition can be moved out to handle this

Reply:
Yes Right have made the change https://github.com/cerner/terra-framework/pull/2031/commits/dbacc7e8f6ee54df11ab377b0bf50949bf093e3e